infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -235,6 +235,30 @@ param parTags object = {}
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry.')
 param parTelemetryOptOut bool = false
 
+@sys.description('Define inbound destination port or a range for HTTPS. Default:443')
+param parHttpsPort string = '443'
+
+@sys.description('Define inbound destination port or a range for gateway manager. Default:443')
+param parGatewayManagerPort string  ='443'
+
+@sys.description('Define inbound destination port or a range for load balancer. Default:443')
+param parLoadBalancerPort string = '443'
```

Comment:
Which load balancer? 

Propose we remove anyway and hardcode as before as this isnt something customers should be able to change